### ISSUE TITLE
The ninja den's items can be brought back with you. Like, anything probably.

### ISSUE THREAD

**Issue by necromanceranne** [tgstation/tgstation#85052]
Reporting client version: 515.1640

## Issue Summary

It's so nice that the Spider Clan have given me this meat hook to avenge my kidnapping with.

![image](https://github.com/user-attachments/assets/83abfd38-0caa-4a6e-a9cc-5c3b1d654fdc)

## Round ID:
232247

## Testmerges:
- [Ghosts once again can see people's true names and roundstart jobs](https://www.github.com/tgstation/tgstation/pull/84951)

## Reproduction:

Step 1: Get kidnapped
Step 2: Stuff your pockets with gamer gear.
Step 3: Meditate or something.
Step 4: Get sent back with free gamer gear.

**Comment by moocowswag** [comment]
Likely bug caused by that pr that lets you take a shirt back as a souvenir?

**Comment by necromanceranne** [comment]
I don't think so. That PR is at least a year old, and I'm pretty sure this is relatively recent. I could not do this a few months ago.

**Comment by zoomachina** [comment]
this is a bug? I'm pretty sure this has been around for months if not years

**Comment by Profakos** [comment]
The shirt PR is not related to this issue. All it did was add the shirt, did not add any new mechanics.
